lrc_file/LrcFile.py

```diff
@@ -203,6 +233,28 @@ def flow_result_data(self) -> dict[str, float]:
         """
         return self._flow_metrics
 
+    @property
+    def flow_evaluation_data(self) -> dict[str, float]:
+        """
+            Returns flow metrics with its values used during evaluation.
+        """
+        evaluation_data = {}
+
+        for result in self._data.results:
+            if not isinstance(result, BaselineEvaluationResult):
+                continue
+
+            for comparison in result.data["comparisons"]:
+                if not isinstance(comparison["current_result"], AggregatedFlowMeasurementResults):
```

Comment:
i think this should use the base type of `FlowMeasurementResults` instead of the aggregated version, just in case there's some use case in the future where only a single measurement is done without repetitions.

Reply:
thanks